members/B001135.yaml

```diff
@@ -4,10 +4,12 @@ contact_form:
   action: /contact/email
   steps:
     - visit: "http://www.burr.senate.gov/contact/email"
+    - find:
+      - selector: div#morestuffbutton
```

Comment:
Is this the time for the `find`? I had thought it wasn't finding `firstname`.

Reply:
So this form is a little different, in that  `div#morestuffbutton` first needs to be clicked for the form to load all of the required inputs. 
[![Screenshot from Gyazo](https://gyazo.com/e4195fa3d710b7b7556f544ecf82410c/raw)](https://gyazo.com/e4195fa3d710b7b7556f544ecf82410c)

I think the `find` up there would help a little bit to make sure that selector is clicked everytime. Now that you point this out, I noticed that the second `find` looks for `select#prefixfocus` but the step after it is a `fill-in: first-name`. 

Let me re-arrange that and move the `- select: prefix` step up so it comes right after that `find`.